dhall/src/Dhall/Main.hs

```diff
@@ -822,9 +822,15 @@ command (Options {..}) = do
                         Dhall.Core.throws (Dhall.Binary.decodeExpression bytes)
 
 
-            let doc = Dhall.Pretty.prettyCharacterSet characterSet (Dhall.Core.renote expression :: Expr Src Import)
+            if quiet
+                then return ()
+                else do
+                    let doc =
+                            Dhall.Pretty.prettyCharacterSet
+                                characterSet
+                                (Dhall.Core.renote expression :: Expr Src Import)
```

Comment:
(style comment) use `unless` (from `Control.Monad`):

```suggestion
            unless quiet $ do
                    let doc =
                            Dhall.Pretty.prettyCharacterSet
                                characterSet
                                (Dhall.Core.renote expression :: Expr Src Import)
```

Reply:
Yeah. When I started contributing to `dhall`, I often wanted to use `unless` and similar utils too, but realized that Gabriel mostly sticks to `if`. These days, I find this `if`-style actually more readable than the implicit negation of `unless`.